tests/testthat/test-plot_ggonion.R

```diff
@@ -6,3 +6,11 @@ testthat::test_that("plot ggonion", {
   clr <- c("red","yellow", "green")
   ggonion(x, ratio = 2, bias = 0, color = clr)
 })
+testthat::test_that("plot ggonion with themes", {
+x <- c("one", "two", "three")
+clr <- c("red","yellow", "green")
+base <- ggonion(x, ratio = 2, bias = 0, color = clr)
+base + theme_grey() + ggtitle("theme_grey()")
```

Comment:
We should use only palettes from themes, no need to draw additional axis, labels etc